ddtrace/contrib/tornado/stack_context.py

```diff
@@ -20,7 +20,7 @@ class TracerStackContext(DefaultContextProvider):
     https://github.com/tornadoweb/tornado/issues/1063
     """
     def __init__(self):
-        super(TracerStackContext, self).__init__()
+        super(TracerStackContext, self).__init__(reset_context_manager=False)
```

Comment:
why do we want `False` here?

we should add a comment, since this goes against defaults/other places.